src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp

```diff
@@ -153,7 +153,7 @@ APPLICATION_INFO::UpdateAppOfflineFileHandle()
             STACK_STRU(strEventMsg, 256);
             if (SUCCEEDED(strEventMsg.SafeSnwprintf(
                 ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_MSG,
-                m_pConfiguration->QueryApplicationPath()->QueryStr())))
```

Comment:
Why change this?

Reply:
`m_pConfiguration->QueryApplicationPath()->QueryStr()` is an empty string. It should have been QueryApplicationFullPath. I'm going to investigate what else is using QueryApplicationPath().

Reply:
the filePath is the same as the application path except it has app_offline in the path.

Reply:
Just change it to QueryApplicationPhysicalPath then